# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1533140235 14400
#      Wed Aug 01 12:17:15 2018 -0400
# Branch stable
# Node ID 035ebdd4f06a619bbfc04d56e6164d1382b62deb
# Parent  439e69ae7aeb759b011198663a538e6dbec96e1d
* run.m: In test, use a symbol name that is unlikely to be a function.

diff --git a/scripts/miscellaneous/run.m b/scripts/miscellaneous/run.m
--- a/scripts/miscellaneous/run.m
+++ b/scripts/miscellaneous/run.m
@@ -86,29 +86,30 @@ function run (script)
     evalin ("caller", sprintf ("source ('%s');", script),
             "rethrow (lasterror ())");
   endif
 
 endfunction
 
 
 ## Test script file execution
+## Use a variable name that is unlikely to be the name of a function.
 %!test
-%! clear A  # the variable "A" should be set by the script
-%! assert (exist ("A"), 0);
+%! clear _5yVNhWVJWJn47RKnzxPsyb_
+%! assert (exist ("_5yVNhWVJWJn47RKnzxPsyb_"), 0);
 %! tmp_dir = tempname ();
 %! test_script = fullfile (tmp_dir, "test_script.m");
 %! unwind_protect
 %!   mkdir (tmp_dir);
 %!   fid = fopen (test_script, "w");
-%!   fprintf (fid, "A = 1337;\n");
+%!   fprintf (fid, "_5yVNhWVJWJn47RKnzxPsyb_ = 1337;\n");
 %!   fclose (fid);
 %!   run (test_script);
-%!   assert (exist ("A", "var"), 1);
-%!   assert (A, 1337);
+%!   assert (exist ("_5yVNhWVJWJn47RKnzxPsyb_", "var"), 1);
+%!   assert (_5yVNhWVJWJn47RKnzxPsyb_, 1337);
 %! unwind_protect_cleanup
 %!   unlink (test_script);
 %!   rmdir (tmp_dir);
 %! end_unwind_protect
 
 ## Test function file execution
 %!test
 %! path_orig = path ();
